client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -1007,6 +1006,7 @@ body.is-section-signup.is-white-signup .is-onboarding-2023-pricing-grid .signup_
 	}
 
 	.plan-comparison-grid__billing-info {
+		flex-basis: 50px;
```

Comment:
The display mode of `.plan-comparison-grid__billing-info` is `block`, so I don't think this will work. Could you help me confirm what we want to acheive here?

Reply:
Ah I believe that might be a missed line I forgot to remove. I'll take care of it now 👍

Reply:
[b6d35af](https://github.com/Automattic/wp-calypso/pull/77039/commits/b6d35af04044a1a51ef9f33ecc9402d20a2e5c31)